manuscript/060_Statement_First_Programming.md

```diff
@@ -1,56 +1,56 @@
 
-By the way, this process is sometimes referred to as �Red-Green-Refactor". I�am just mentioning it here for the record -- I�am not planning to use this term further in the book.
+By the way, this process is sometimes referred to as “Red-Green-Refactor". I am just mentioning it here for the record -- I'll not use this term further in the book.
 
 "Test-First" means seeing a failure
 -----------------------------------
 
-When I showed you a drawing with a TDD process, I specifically said that you are supposed to write a Statement that you wish was true **but is not**. It means that when you write a�Statement, you have to evaluate it (i.e. run it) and watch it fail its assertions before providing implementation that makes this Statement true. Why is that so important? Is it not just enough to write the Statement first? Why run it and watch it fail?
+Explaining the illustration with the TDD process above, I pointed out that you are supposed to write a Statement that you wish were true **but is not**. It means that when you write a Statement, you have to evaluate it (i.e. run it) and watch it fail its assertions before you provide an implementation that makes this Statement true. Why is that so important? Is it not just enough to write the Statement first? Why run it and watch it fail?
 
-There are multiple reasons and I�will try to outline few of them briefly.
+There are several reasons and I will try to outline a few of them briefly.
 
 ### You do not know whether the Statement can ever be false until you see it evaluate as false
 
-Every accurate Statement (do I�have to tell you that such Statements are what we are interested in?) fails when it isn�t fulfilled and passes when it is. That is one of the main reasons we write it -- to receive this feedback. Also, after being fulfilled, the Statement becomes a�part of the executable specification and starts failing as soon as the code stops fulfilling it (e.g. as a�result of mistake made during code rework). When your run a�Statement after it is implemented and it is evaluated as true, how do you know whether it really describes a�need accurately? You did not ever watch it fail, so how do you know it ever will?
+Every accurate Statement fails when it isn’t fulfilled and passes when it is. That is one of the main reasons why we write it -- to receive this feedback. Also, after being fulfilled, the Statement becomes a part of the executable specification and starts failing as soon as the code stops fulfilling it, for example as the result of a mistake made during code rework. If you run a Statement for the first time only after it has been implemented and it evaluates to true, how do you know whether it really describes a need accurately? You did not ever watch it fail, so how do you know it ever will?
```

Comment:
Could you also fix this one: "If you run a Statement for the first time only after it has been implemented and it evaluates to true". Looking at it now, I read it as if I meant implementing the code that makes the specification, not implementing what the specification describes.

maybe something like "If you run a Statement for the first time only after the behavior it describes has been implemented and it evaluates to true" ..?